modules/swagger-codegen/src/main/resources/typescript-jquery/typings.mustache

```diff
@@ -3,6 +3,7 @@
         "bluebird": "registry:dt/bluebird#2.0.0+20160319051630",
         "core-js": "registry:dt/core-js#0.0.0+20160317120654",
         "node": "registry:dt/node#4.0.0+20160423143914"
+        /* TODO: Here, we need the typings for jquery... */
```

Comment:
Here, I tried using the "typings" command line but kept receiving super weird error. In anyone can help find the good typings for jquery so the code can finally transpiled, it would be appreciated! :)